acceptance_tests/test_register_page.py

```diff
@@ -1,48 +1,45 @@
 from urlparse import urljoin
-from django.test import TestCase
 
+from django.core.urlresolvers import reverse
 from page_objects import PageObject, PageElement
-from selenium import webdriver
+
+from mixins import BaseTestCase
+from users.tests.mixins import CreateUserMixin
 
 
 class RegistrationPage(PageObject):
     username = PageElement(name='username')
     password = PageElement(name='password1')
     confirmation_password = PageElement(name='password2')
-    registration_button = PageElement(class_name='submit-register')
+    registration_button = PageElement(class_name='login-register-button')
 
 
-class RegisterPageValidation(TestCase):
-    HOST = 'http://127.0.0.1:8080/'
+class RegisterPageValidation(CreateUserMixin, BaseTestCase):
+    register_url = reverse('users:register')
 
     def setUp(self):
-        self.driver = webdriver.Chrome()
-        self.driver.get(urljoin(self.HOST, 'users/register'))
+        super(RegisterPageValidation, self).setUp()
+        self.driver.get(urljoin(self.live_server_url, self.register_url))
         self.page = RegistrationPage(self.driver)
 
-    def test_user_submitted_existing_username(self):
-        self.page.username = 'test'
-        self.page.password = 'pass1234'
-        self.page.confirmation_password = 'pass1234'
+    def fill_registration_form(self, username, password, confirmation_password):
```

Comment:
Make `confirmation_password` an optional parameter and use password when it's not provided, in case you need both to exist.